motoko/token_transfer_from/README.md

```diff
@@ -250,15 +232,15 @@ actor {
 
 ```
 
-### Step 8: Deploy the token transfer canister:
+## Step 9: Deploy the token transfer canister
 
 ```bash
 dfx deploy token_transfer_from_backend
 ```
 
-### Step 9: Approve the canister to transfer funds on behalf of the user:
+## Step 10: Approve the canister to transfer funds on behalf of the user
 
-:::info 
+:::info
 
 Make sure that you are using the default `dfx` account that we minted tokens to in step 5 for the following steps.
```

Comment:
```suggestion
Make sure that you are using the default `dfx` account that we minted tokens to in step 6 for the following steps.
```

I am a little bit confused which step is correct after looking at the other PR. but I think step 6 is correct here 🤔